contracts/SpokePool.sol

```diff
@@ -42,21 +42,21 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
     address public hubPool;
 
     // Timestamp when contract was constructed. Relays cannot have a quote time before this.
-    uint64 public deploymentTime;
+    uint32 public deploymentTime;
 
     // Any deposit quote times greater than or less than this value to the current contract time is blocked. Forces
-    // caller to use an up to date realized fee.
-    uint64 public depositQuoteTimeBuffer;
+    // caller to use an up to date realized fee. Defaults to 10 minutes.
+    uint32 public depositQuoteTimeBuffer = 600;
 
     // Use count of deposits as unique deposit identifier.
-    uint64 public numberOfDeposits;
+    uint32 public numberOfDeposits;
```

Comment:
nit: can we move these down below weth, since I assume weth _and_ all of these will be read on each deposit, so we'll end up saving one word's worth a reads by grouping them.

Reply:
nice done